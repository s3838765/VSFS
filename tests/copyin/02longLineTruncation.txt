Hello this is a long line in an external text file. This will be used for testing the copyin functionality for a file system. Copying this file in should truncate this line at 255 characters; the pre-defined maximum. If the line is not long (i.e. not 255+ characters), it will not be truncated. The next line will be shorter than 255 characters to demonstrate this.
This is a shorter line :)
