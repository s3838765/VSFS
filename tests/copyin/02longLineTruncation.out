NOTES V1.0
=copyin/
@copyin/02longLineTruncation
 Hello this is a long line in an external text file. This will be used for testing the copyin functionality for a file system. Copying this file in should truncate this line at 255 characters; the pre-defined maximum. If the line is not long (i.e. not 255
 This is a shorter line :)
